frontend/src/pages/modelServing/screens/metrics/SPDTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Stack, StackItem, Tooltip } from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const SPDTooltip = () => (
```

Comment:
Are these tooltips used more than once? You are creating a component when it is only used once, and that component is very similar to DIRTooltip. As to not have a lot of unnecessary react components, i would either make the tooltip more generic, or just add this tooltip code to wherever its used

let me know if this was not clear

Reply:
Good point. How about this:

- The generic code for the tooltip could live in the TrustyChart.
- Add an optional prop to trusty chart to take a ReactElement containing the specific text
- The actual text could be defined in DIRChart / SPDChart and passed through

This would eliminate 2 x single use components and would do a better job of keeping the responsibilities in the right places, i.e. layout and logic in the generic TrustyChart component and the specific text in the specialised Chart components.

WDYT?

Reply:
@Gkrumbach07 ^

Reply:
that is what i was thinking